src/cfnlint/data/ExtendedSpecs/us-west-2/05_pricing_property_values.json

```diff
@@ -91,6 +91,7 @@
       "i3en.6xlarge",
```

Comment:
https://github.com/aws-cloudformation/cfn-python-lint/pull/1535/ seems to be working 🎉